qutebrowser/mainwindow/statusbar/bar.py

```diff
@@ -215,6 +213,46 @@ def _on_config_changed(self, option):
             self.maybe_hide()
         elif option == 'statusbar.padding':
             self._set_hbox_padding()
+        elif option == 'statusbar.widgets':
+            self._draw_widgets()
+
+    def _draw_widgets(self):
+        """Draw statusbar widgets"""
+        # Start with widgets hidden and show them when needed
+        for widget in [self.percentage, self.backforward, self.tabindex]:
```

Comment:
I wonder if there's a cleaner way of removing these. Looking at the `QLayout` class, there are methods to get all the widgets (`itemAt` and `count`), but those include all the widgets, and not just the ones we are interested in. I'm not sure if it would be cleaner as it is now or with a filtering system ontop of the QLayout....

We might be able to put another `HBox` in here, to hold nothing but widgets, but that would be a bigger rewrite.